Sources/Public/Animation/DotLottie/ZIPFoundation/Archive+BackingConfiguration.swift

```diff
@@ -0,0 +1,128 @@
+//
+//  Archive+BackingConfiguration.swift
+//  ZIPFoundation
+//
+//  Copyright © 2017-2021 Thomas Zoechling, https://www.peakstep.com and the ZIP Foundation project authors.
+//  Released under the MIT License.
+//
+//  See https://github.com/weichsel/ZIPFoundation/blob/master/LICENSE for license information.
```

Comment:
I see you've included the source of ZIPFoundation directly. I think it would be preferable to incorporate ZIPFoundation through a package manager. Have you explored that approach?